### PR TITLE
[Bugfix][Printer] Avoid adding annotation twice for ConstantNode

### PR DESCRIPTION
**Pull request by yzhliu**
@icemelon9 @zhiics @jroesch 

the printer of constant node prints annotation (and typer) twice. like,

conv2d(%x, %conv.w, meta[relay.Constant][0] /* ty=Tensor[(1), int64] */ /* ty=Tensor[(1), int64] *

Because for Print(meta=false), constant node does not know how to print itself, and fallback to Print(meta=true), the annotation therefore be added twice in Print().

### CODE REVIEW

File: src/printer/relay_text_printer.cc [hunk 1]
```diff
@@ -330,6 +330,14 @@ Doc RelayTextPrinter::VisitExpr_(const ConstantNode* op) {
       return ScalarLiteral(dtype, static_cast<const uint8_t*>(op->data->data)[0]);
     }
   }
+  const PackedFunc* fprint = runtime::Registry::Get("relay._constant_text");
```
Comment (zhiics) [inline src/printer/relay_text_printer.cc:333]: Is `relay._constant_text ` defined by users?

Comment (junrushao1994) [reply src/printer/relay_text_printer.cc:333]: I think it is a global function that third party can plug in and customize the printing for constants

Comment (zhiics) [reply src/printer/relay_text_printer.cc:333]: Yeah, I am okay with this. I think another possible way could use a user-defined annotator and pass it to the pretty printer. One example could be something like:

```python
def constant_annotator():
    def _annotate(e):
        if isinstance(e, relay.Constant):
            # Print the constant
        else:
            return ""
    return _annotate

# Given an IRModule m or an expression m
print(m.astext(show_meta_data=False, annotate=constant_annotator()))
```

**Comment by yzhliu** [comment]
Thanks @zhiics @junrushao1994 for the suggestion. The annotator does work. However I noticed the printer of constant node prints annotation (and typer) twice. like,

```
conv2d(%x, %conv.w, meta[relay.Constant][0] /* ty=Tensor[(1), int64] */ /* ty=Tensor[(1), int64] *
```

Because for Print(meta=false), constant node does not know how to print itself, and fallback to Print(meta=true), the annotation therefore be added twice in `Print()`.

Please advise if the fix makes sense.

**Comment by zhiics** [comment]
@yzhliu yeah, I noticed this before as well. I am okay with the change.

**Review by zhiics — APPROVED** [review]

**Comment by zhiics** [comment]
Thanks @yzhliu @junrushao1994

**Pull request merged**